source/dbo/Stored Procedures/Reusable Notification/uspGenerateReusableNotificationTreatmentRegimen_ETS.sql

```diff
@@ -0,0 +1,43 @@
+﻿/***************************************************************************************************
+Desc:    The ETS TreatmentPlanned table contains two fields 'ShortCourse' and 'MDRTreatment'.
+There is only one field in ReusableNotification to capture this information, so set this field based
+on the values of these two input fields.
+
+The three possible values for ReusableNotification.TreatmentRegimen come from the same field in
+NTBS's ClinicalDetails table, and are:
+
+		StandardTherapy,
+		MdrTreatment,
+		Other
+**************************************************************************************************/
+
+
+CREATE PROCEDURE [dbo].[uspGenerateReusableNotificationTreatmentRegimen_ETS]
+
+AS
+
+	UPDATE dbo.ReusableNotification_ETS
+		SET TreatmentRegimen = 'StandardTherapy'
```

Comment:
As comment above - this needs to output proper phrases, not the 'code' values used in the application.  Unfortunately this enum is held in code only and doesn't live in the ReferenceData schema in NTBS, where we could grab the description from the code. What I've done for other similar issues in the reporting database is create a lookup table which will need to be maintained in sync with the code, e.g. https://github.com/publichealthengland/ntbs-reporting/blob/master/source/dbo/Tables/Reusable%20Notification/OutcomeLookup.sql